src/App.tsx

```diff
@@ -1,30 +1,157 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import React, { useEffect, useState } from 'react';
+import { Todo } from './types/Todo';
+import { addTodo, deleteTodo, getTodos, updateTodo } from './api/todos';
+import { wait } from './utils/fetchClient';
+import { Todo as TodoElement } from './components/Todo';
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+enum ERROR_MESSAGE {
+  serverError = 'Unable to load todos',
+  emptyError = 'Title should not be empty',
+  addError = 'Unable to add a todo',
+  deleteError = 'Unable to delete a todo',
+  updateError = 'Unable to update a todo',
+}
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  const [todoData, setTodoData] = useState<Todo[]>([]);
+  const [filteredData, setFilteredData] = useState<Todo[]>([]);
+  const [fillterStatus, setFilterStatus] = useState('all');
+  const [todoCounter, setTododCounter] = useState<number>(0);
+
+  // const [loading, setLoading] = useState(false);
+
+  const [showErrorBox, setShowErrorBox] = useState(false);
+
+  const [textOfError, setTextOfError] = useState<ERROR_MESSAGE | null>(null);
+  const [todoInput, setTodoInput] = useState('');
+
+  useEffect(() => {
+    getTodos()
+      .then(data => {
+        setTodoData(data);
+      })
+      .catch(() => {
+        setShowErrorBox(true);
+        setTextOfError(ERROR_MESSAGE.serverError);
+      })
+      .finally(() => {});
+  }, []);
+
+  function addPost(title: string) {
+    const ownData = {
+      userId: 690,
+      title: title,
+      completed: false,
+    };
+
+    addTodo(ownData)
+      .then((data: any) => {
+        setTodoInput('');
+        setTodoData((prevData: Todo[]): any => {
```

Comment:
Fix it 
<img width="597" alt="image" src="https://github.com/user-attachments/assets/aeea4f31-be50-42f8-947c-65d9473dfe9c">